resources/bch/servers.json

```diff
@@ -144,5 +144,11 @@
         "s": "50002",
         "t": "50001",
         "version": "1.4.4"
+    },
+    "fulcrum.apecs.dev": {
+        "pruning": "-",
+        "s": "50002",
+        "t": "50001",
+        "version": "1.9.0"
```

Comment:
The version is protocol version not server version. Should be `1.5.0` or something.